pkg/kev/manifest.go

```diff
@@ -34,6 +34,10 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// Infix component of generated docker-compose filenames
+// (e.g. results in docker-compose.env.dev.yaml)
+const dockerComposeInfix = "env"
```

Comment:
As this infix is only ever used in the environment override files how about renaming this const to `envOverrideFileInfix`?

Reply:
Good call, done!